### PR TITLE
{math,phys}[foss/2019a,intel/2019a] kwant v1.4.1 w/ Python 3.7.2, MUMPS v5.2.1 sequential

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1845~~

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/225f7f834d8c353f033f3eb35fd324ab for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 0 out of 4 (4 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/5e88dff6b17efb6ce7bf75e665162a08 for a full test report.

File: easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1-foss-2019a-metis-seq.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name = 'MUMPS'
+version = '5.2.1'
+versionsuffix = '-metis-seq'
+
+homepage = 'https://graal.ens-lyon.fr/MUMPS/'
+description = "A parallel sparse direct solver"
```
Comment (akesandgren) [inline easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1-foss-2019a-metis-seq.eb]: Perhaps add that this is a serial version to the description?

File: easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1-intel-2019a-metis-seq.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name = 'MUMPS'
+version = '5.2.1'
+versionsuffix = '-metis-seq'
+
+homepage = 'https://graal.ens-lyon.fr/MUMPS/'
+description = "A parallel sparse direct solver"
```
Comment (akesandgren) [inline easybuild/easyconfigs/m/MUMPS/MUMPS-5.2.1-intel-2019a-metis-seq.eb]: Same here.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node3124.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/19fa4c861d9a013b58d516bc7d2f95eb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in this PR)
node2452.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/242ff8eb828436ca44d03404f9271bcb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2568.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a63267627fffdcee33abbd51f368c0fa for a full test report.

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**